### ISSUE TITLE
[RemoteConfig] default values missing/excluded from keysWithPrefix/allKeysFromSource

### ISSUE THREAD

**Issue by Salakar** [firebase/firebase-ios-sdk#3092]
### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 10.1
  * Firebase SDK version: 5.20.2
  * Firebase Component: RemoteConfig (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: 5.20.2

### [REQUIRED] Step 3: Describe the problem

Hey 👋 

I may be doing something dumb but;

When using any of the following methods

```obj-c
- (void)setDefaults:(nullable NSDictionary<NSString *, NSObject *> *)defaults;

- (void)setDefaultsFromPlistFileName:(nullable NSString *)fileName;
```

it's not possible to view these default values unless you know all the key names. On Android calling `getAll` it includes all the `default source` values merged alongside the `remote source` values. However; on iOS (via `keysWithPrefix:nil`) only `remote source` values are shown with `default source` values not merged in.

I figured maybe I have to manually get these via `allKeysFromSource` myself, e.g.:

```obj-c
  NSArray<NSString *> *staticKeys = [[FIRRemoteConfig remoteConfig] allKeysFromSource:FIRRemoteConfigSourceStatic namespace:nil];
  NSArray<NSString *> *defaultKeys = [[FIRRemoteConfig remoteConfig] allKeysFromSource:FIRRemoteConfigSourceDefault namespace:nil];
```

![image](https://user-images.githubusercontent.com/5347038/58432943-d05d0c80-80ac-11e9-8378-3ee96cc1993a.png)

but both of these return empty arrays as shown above; so I'm unable to enumerate these in any form. To make sure default values are being set, I inspected `FIRRemoteConfig` via the debugger:

**Defaults are being set internally**:
![image](https://user-images.githubusercontent.com/5347038/58433016-12864e00-80ad-11e9-9ef5-f6e02f6285fb.png)

**Active config excludes default values**:
![image](https://user-images.githubusercontent.com/5347038/58433061-4497b000-80ad-11e9-963a-e811001ef19c.png)

TLDR; unless you know all the default key names then there's no way to retrieve them? This is different behaviour than on Android and is limiting the implementation of Remote Config support in React Native Firebase as these need to be gathered and then transferred into JavaScript

#### Steps to reproduce:

```obj-c
[[FIRRemoteConfig remoteConfig] setDefaults:@{
    @"testing": @"a default value"
}];
  
NSSet *keys = [[FIRRemoteConfig remoteConfig] keysWithPrefix:nil];
// 'testing' will not exist in set

NSArray<NSString *> *defaultKeys = [[FIRRemoteConfig remoteConfig] allKeysFromSource:FIRRemoteConfigSourceDefault namespace:nil];
// 'testing' will not exist in array
```

**Comment by Salakar** [comment]
Looks like this was me, though the docs aren't very clear on this but the namespace for `allKeysFromSource` has to be specified even though it accepts nullable (I assumed nil used the default namespace) but a namespace of  `FIRNamespaceGoogleMobilePlatform` is required.

**Comment by ryanwilson** [comment]
Thanks for the comment and pointing out the namespace issue! Having it `nullable` definitely isn't clear. We'll have to take a look and see what we can do to improve the docs and the API.
